tools/gen_docs.py

```diff
@@ -106,7 +106,7 @@ def hash_files(verbose, debug):
             h.update(f.read().replace(b'\r\n', b'\n'))
     result = h.hexdigest()
     print('HASH: ', result)
-    return result
+        return result
```

Comment:
This breaks linting. Return should have indention aligned with everything else under the same scope.